packages/services/src/vckit.service.ts

```diff
@@ -150,3 +153,41 @@ const constructCredentialObject = ({ context, type, issuer, credentialSubject, .
     },
   };
 };
+
+/**
+ * Integrate with vckit to verify VC
+ * @param verifiableCredential
+ * @param vcKitAPIUrl
+ * @returns
+ */
+export const verifyVC = async (
+  verifiableCredential: VerifiableCredential,
+  vcKitAPIUrl?: string,
+): Promise<IVerifyResult> => {
+  const verifyCredentialParams = {
+    credential: verifiableCredential,
+    fetchRemoteContexts: true,
+    policies: {
+      credentialStatus: true,
+    },
+  };
+
+  privateAPI.setBearerTokenAuthorizationHeaders(appConfig.defaultVerificationServiceLink.apiKey ?? '');
```

Comment:
I have an error with this code:
<img width="816" alt="image" src="https://github.com/user-attachments/assets/1a0de41c-67e5-4501-a2f9-2c16eb86104b">

Reply:
Fixed in the latest commit.